Configuration/ToolboxArgumentParser.cs

```diff
@@ -1,4 +1,5 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
 using Microsoft.Extensions.CommandLineUtils;
 
 namespace QuantConnect.Configuration
```

Comment:
Missing doc entry for `randomdatagenerator` [here](https://github.com/mchandschuh/Lean/blob/32f7704b18351f1f7ef6a4f2b7e7e4712ce02ebf/Configuration/ToolboxArgumentParser.cs#L26)